src/main/java/com/serhiiklymchuk/gmailpet/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package com.serhiiklymchuk.gmailpet.service;
+
+import com.serhiiklymchuk.gmailpet.domain.User;
+import com.serhiiklymchuk.gmailpet.dto.UserDto;
+
+public interface UserService {
```

Comment:
use right spacing.
Empty line before the first field or method in class, lines between them, line after.
Revise all the places in you PR.